modular-docs-manual/files/TEMPLATE_REFERENCE_reference-material.adoc

```diff
@@ -42,3 +42,13 @@ Term 2:: Definition
 |Row 1, column 1|Row 1, column 2|Row 1, column 3
 |Row 2, column 1|Row 2, column 2|Row 2, column 3
 |====
+
+[role="_additional-resources"]
+.Additional resources
+////
+Optional. Delete if not used.
+////
+* A bulleted list of links to other material closely related to the contents of the concept module.
+* Currently, modules cannot include xrefs, so you cannot include links to other content in your collection. If you need to link to another assembly, add the xref to the assembly that includes this module.
+* For more details on writing concept modules, see the link:https://github.com/redhat-documentation/modular-docs#modular-documentation-reference-guide[Modular Documentation Reference Guide].
```

Comment:
@emmurphy1 Sounds good to me. Though I assume since this bit is copied-pasted, we probably want this to say "For more details on writing reference modules, ..." Right? Otherwise good with me, so approving assuming that minor tweaks is addressed.

Reply:
Good catch. Fixed.